src/puppetlabs/puppetdb/query.clj

```diff
@@ -120,14 +120,14 @@
   whose value is an integer indicating the total number of results available."
   ([query paging-options] (execute-query 0 query paging-options))
   ([fail-limit query {:keys [limit offset order_by] :as paging-options}]
-     {:pre [((some-fn string? sequential?) query)]
-      :post [(map? %)
-             (vector? (:result %))
-             ((some-fn nil? integer?) (:count %))]}
-     (let [sql-and-params (if (string? query) [query] query)]
-       (if (requires-paging? paging-options)
-         (execute-paged-query* fail-limit sql-and-params paging-options)
-         (execute-query* fail-limit sql-and-params)))))
+   {:pre [((some-fn string? sequential?) query)]
+    :post [(map? %)
+           (vector? (:result %))
+           ((some-fn nil? integer?) (:count %))]}
+   (let [sql-and-params (if (string? query) [query] query)]
+     (if (requires-paging? paging-options)
+       (execute-paged-query* fail-limit sql-and-params paging-options)
+       (execute-query* fail-limit sql-and-params)))))
 
```

Comment:
Not sure where we ended up with respect to whitespace, but this appears incidental (if you were trying to avoid it now).  And it looks like there's more in the rest of the patch.